slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/dao/DatabaseVersionDao.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.slashingprotection.dao;
+
+import org.jdbi.v3.core.Handle;
+
+public class DatabaseVersionDao {
+
+  public Integer findDatabaseVersion(final Handle handle) {
+    try {
+      return handle
+          .createQuery("SELECT version from database_version WHERE id = 1")
```

Comment:
just for clarification ... so there will always be one record and version field would keep updating? Rather than inserting a new record every time we update and pick the highest (or latest) version.